docs/man/containerd-config.toml.5.md

```diff
@@ -167,48 +168,242 @@ the main config.
 The following is a complete **config.toml** default configuration example:
```

Comment:
Perhaps I misunderstand this to read the output of `containerd config default`, should this instead be a small example? (default - empty values?)